v3/src/components/free-tile-row.scss

```diff
@@ -10,7 +10,7 @@
     position: absolute;
     box-shadow: 0 1px 3px 0 rgb(0 0 0 / 25%);
     border-radius: 6px;
-    transition: height 0.25s;
+    transition: height 0.5s ease-in-out 0s;
```

Comment:
In V2 the duration of the transition is 0.4s. Unless there's a reason, let's use that.

Reply:
The slider has its own `transition` statement. Whatever values are decided upon, it should probably be put in `vars.scss` so it's not duplicated in multiple places.